ui/src/style/pdf-viewer.less

```diff
@@ -21,7 +21,8 @@
   * Consistent highlight between default pdf.js highlight for tables, figures, etc. 
   * and custom purple highlight for citation annotations.
   */
-.linkAnnotation a:not([href^="#cite"]):hover {
+.annotationLayer .linkAnnotation > a:hover,
+.annotationLayer .buttonWidgetAnnotation.pushBotton > a:hover {
```

Comment:
@jocelynzsun When does this second selector (`.annotationLayer .buttonWidgetAnnotation.pushBotton > a:hover`) become necessary? Can we just use `.annotationLayer .linkAnnotation > a:hover`?

Reply:
We are probably not using the feature it applies to seeing that everything still behaves the same way when I delete it. it seems like `.annotationLayer .linkAnnotation > a:hover` is sufficient